packages/video_player/video_player/lib/video_player.dart

```diff
@@ -106,6 +107,9 @@ class VideoPlayerValue {
   /// The current speed of the playback.
   final double playbackSpeed;
 
+  /// True if picture-in-picture is currently active.
+  final bool isPictureInPictureActive;
```

Comment:
@vanlooverenkoen 
Nice job, I'm in need of this feature.

I'm testing this pr in a project and I found a bug when updating this value. This property is not used in the `bool operator ==(Object other)` function, so it is not updated in the ValueNotifier. Can you fix this?

Reply:
Damn! Good catch!

Reply:
It doesn't look like you added a Dart unit test to cover that change (i.e., one which would have caught this bug).

Reply:
> It doesn't look like you added a Dart unit test to cover that change (i.e., one which would have caught this bug).

This doesn't appear to have been resolved.